### ISSUE TITLE
[Bug]: Fullscreen cuts off page nav in deployed environment

### ISSUE THREAD

**Issue by mdp1125** [evidence-dev/evidence#2447]
### Describe the bug

Seems to be related to the resolution of issue https://github.com/evidence-dev/evidence/issues/2144 from this PR https://github.com/evidence-dev/evidence/pull/2308 in v39.1.4.

The PR and issue resolution works great in local (results dynamically size and page nav bar remains set at bottom).

However, in deployed environment (app.supplyscope.ai), it does not function the same way.

When opening the fullscreen, the results can dynamically size as the window changes but when the results exceed some set number, the page nav bar is cut off, which disrupts our user workflow.

My browser user agent is this using Google Chrome
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36

Im on a Macbook Pro 14 Inc Apple M3 Pro chip on MacOS Sonoma 14.1. I have provided the system info from my local below, however it seems to work great when in the local dev mode.

I can't show much on this issue as the data is proprietary but this is what the bottom of the fullscreen mode looks like when there's over 800 results. Nav page is nowhere to be seen
![image](https://github.com/user-attachments/assets/ab03d11a-1f0f-4e9f-8c7e-b0b2da56dc51)

### Steps to Reproduce

Log in to our deployed environment and see if you have the same issue?

### System Info

```Shell
System:
    OS: macOS 14.1
    CPU: (11) arm64 Apple M3 Pro
    Memory: 718.98 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.12.0 - /usr/local/bin/node
    npm: 10.5.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 127.0.6533.120
    Safari: 17.1
  npmPackages:
    @evidence-dev/bigquery: ^2.0.4 => 2.0.5 
    @evidence-dev/core-components: ^4.7.5 => 4.7.5 
    @evidence-dev/csv: ^1.0.7 => 1.0.9 
    @evidence-dev/databricks: ^1.0.3 => 1.0.5 
    @evidence-dev/duckdb: ^1.0.7 => 1.0.9 
    @evidence-dev/evidence: ^39.1.4 => 39.1.4 
    @evidence-dev/mssql: ^1.0.5 => 1.0.7 
    @evidence-dev/mysql: ^1.1.0 => 1.1.2 
    @evidence-dev/postgres: ^1.0.4 => 1.0.5 
    @evidence-dev/snowflake: ^1.1.1 => 1.1.1 
    @evidence-dev/sqlite: ^2.0.3 => 2.0.4 
    @evidence-dev/trino: ^1.0.4 => 1.0.5 
    vite: ^5.2.10 => 5.2.10
```

### Severity

annoyance

**Comment by kwongz** [comment]
Hi @mdp1125 Thanks for bringing this up, will look into this!

**Comment by kwongz** [comment]
fixed by #2448
